api/server/src/main/java/jakarta/websocket/server/HandshakeRequest.java

```diff
@@ -47,8 +47,9 @@
     static String SEC_WEBSOCKET_EXTENSIONS = "Sec-WebSocket-Extensions";
 
     /**
-     * Return the read only Map of Http Headers that came with the handshake request. The header names are case
-     * insensitive.
+     * Return the read only map of Http headers to header values that came with the handshake request. The original case
```

Comment:
Again, I think we should drop the "original case will be preserved" rule / section.

Reply:
Fair enough. I'll drop the "original case" aspects of this.